go/api/base_client.go

```diff
@@ -476,14 +501,21 @@ func (client *baseClient) ResetConnectionPassword() (string, error) {
 //
 // Return value:
 //
-//	`"OK"` response on success.
+//		`"OK"` response on success.
+//
+//	  In transaction:
+//	   It adds the command to the transaction queue instead of executing immediately.
+//	   Upon transaction completion with Exec(), the actual string value will be accessible in the returned results array.
```

Comment:
Just an FYI, developers using `api.NewGlideClient` or `api.NewGlideClusterClient` will not see these comments since those constructors return interfaces, not the structs that embed `baseClient`. Only developers using `api.NewTransaction` will see these because they'll end up getting the `Transaction` struct that embeds `baseClient`.